### ISSUE TITLE
Homework 09 is ready for grading

### ISSUE THREAD

**Issue by KateJohnson** [KateJohnson/STAT545-hw-Johnson-Kate#8]
The repo for my powers package is [here](https://github.com/KateJohnson/powers)

**Comment by Mathnstein** [comment]
Hi @KateJohnson 
It definitely seems like you completed the power package by adding in the vignettes and the readme! I think it could have been cool to have added maybe an inverse or something a bit outside of what we had discussed in class for a function, but everything is well documented!

I did enjoy the additional help for downloading and installing your package, the tutorial for your work was all inclusive!

Overall, maybe just a bit more depth or breadth of functionality, but the point of the assignment was to develop the other components of the package design which I feel you did just well!

Cheers,
Cody

**Comment by yuanjisun** [comment]
Hello @KateJohnson,

This is a great work. Here are my comments.

1. I can install your package successfully.
2. I run the check and there is no error. The `testthat()` function also works and all tests are passed without any problems.
3. You added package documentation and help info for functions. Several methods such as README and vignettes are used properly.
4. All functions are exported successfully.
5. For test files, you check whether the function works under different conditions. Great use of `expect_identical`.
6. I run the function and they all worked!

**Suggestion**
It may be good to test some fail examples. For example, if you input a character, what will happen? There are a lot of `expect_XXX()` function and you can find the one to be used.

In conclusion, great work!

Yuanji
